README.md

```diff
@@ -303,6 +303,36 @@ Any custom metadata fields will be added to the root of the message, so that you
 }
 ```
 
+You can also choose to log caught exceptions with a custom message.
```

Comment:
I think this is a bit too verbose for repo's main `README.me` to include full documentation for all formatters, so let's not add it here.

Reply:
Instead, we should add `:crash_reason` [here](https://github.com/Nebo15/logger_json/blob/master/lib/logger_json.ex#L72) as well-known metadata key supported across loggers, the documentation can look something like this:

```
* `:crash_reason` - accepts a tuple where the first element is the exception
    struct and the second is the stacktrace. For example: `Logger.error("Exception!", crash_reason: {e, __STACKTRACE__})`.
    Each formatter may encode it differently,
```

All the formatters should already support that.

Reply:
Alright understandable, I've removed it from the ReadMe and added that `:crash_reason` documentation to `logger_json.ex`